store/index.js

```diff
@@ -146,6 +149,54 @@ export const getters = {
         return p_state.categories[p_category].componentName;
     },
 
+    getCategoricalJsonOutput: (p_state) => (p_columnName) => {
+
+        // 0. Initialize output object
+        const annotatedDictColumn = p_state.dataDictionary.annotated[p_columnName];
+        const category = p_state.columnToCategoryMap[p_columnName];
+        const formattedOutput = {
+
+            Annotations: {
+
+                IsAbout: {
+
+                    Label: "",
+                    TermURL: ""
+                },
+                Levels: {},
+                MissingValues: []
+            }
+        };
+
+        // 1. Fill out Annotations 'IsAbout' section
+
+        // A. Label matches the assigned category
+        formattedOutput.Annotations.IsAbout.Label = category;
```

Comment:
This is correct! I'm just thinking: now that we have an `identifier` attribute for each category, should we use it to refer to the category (like we're doing in other cases for mappings)? Definitely not for this PR, but maybe worth an issue to discuss.